x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/rule_execution_logic/configs/serverless.config.ts

```diff
@@ -17,6 +17,7 @@ export default createTestConfig({
       '/testing_regex*/',
     ])}`, // See tests within the file "ignore_fields.ts" which use these values in "alertIgnoreFields"
     `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+      'entityAnalyticsAssetCriticalityEnabled',
```

Comment:
Do these flags actually apply in serverless? If not, they should be deleted. If they _do_, but serverless doesn't provide an actual way to set these in production, then they should also be deleted.

Reply:
But as I understand those tests will run in a serverless environment anyway, which allows us to test and integrate our functionality during development, and catch some serverless issues earlier.

If we remove the flag and skip tests, then in time when we enable it and try to run it serverless we can have some problems. For example we didn't have serverless tests for risk engine enablement, and then there were errors for permissions